examples/development-minio/pvc.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: minio
-  namespace: thanos
```

Comment:
I think we can remove this line as other minio manifests don't have namespace specified as well.